deps/rabbit/apps/rabbitmq_prelaunch/src/rabbit_logger_std_h.erl

```diff
@@ -51,7 +51,7 @@
 -export([log/2, adding_handler/1, removing_handler/1, changing_config/3,
          filter_config/1]).
 
--define(DEFAULT_CALL_TIMEOUT, 5000).
+-define(DEFAULT_CALL_TIMEOUT, 15000).
```

Comment:
This module is a fork of `logger_std_h` to add and stabilise date-based file rotation. The goal is to submit the change upstream once we are happy with it.

Therefore I would like this module to remain as close as possible to the original source and thus I'm going to revert this change.